components/TagList.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, FlatList } from 'react-native';
+import { Text } from 'react-native';
+
+export default class TagList extends Component<Props> {
```

Comment:
Would prefer this still be turned into a functional/presentational component - in the future this will likely be either used directly (i.e. in schedule card) or wrapped in a component with state (the toggling menu)
Should be a quick fix